compute_endpoint/globus_compute_endpoint/cli.py

```diff
@@ -910,6 +911,22 @@ def self_diagnostic(compress: bool, log_kb: int):
         click.echo(f"Successfully created {filename}")
 
 
+@app.command(
+    "python-exec",
+    context_settings=dict(ignore_unknown_options=True, allow_extra_args=True),
+)
+@click.argument("module")
+@click.help_option("-h", "--help")
+@click.pass_context
+def run_python_executable(ctx: click.Context, module: str):
+    """Run a Python module as a script via the Globus Compute endpoint CLI.
+    This helps to minimize PATH issues when launching workers, etc.
+
+    E.g., globus-compute-endpoint python-exec path.to.module --opt ahoy
```

Comment:
``--opt ahoy`` is fun but not quite as clear these are user options.  I'd do ``--some_option --ahoy matey`` or something, but this is a pretty minor preference